core/server/common/src/main/java/alluxio/rocks/RocksStore.java

```diff
@@ -84,10 +84,11 @@ public final class RocksStore implements Closeable {
    * @param dbOpts the configured RocksDB options
    * @param columnFamilyDescriptors columns to create within the rocks database
    * @param columnHandles column handle references to populate
+   * @param reset if true, the RocksDB will be cleaned up and then new DB will be created
    */
   public RocksStore(String name, String dbPath, String checkpointPath, DBOptions dbOpts,
       Collection<ColumnFamilyDescriptor> columnFamilyDescriptors,
-      List<AtomicReference<ColumnFamilyHandle>> columnHandles) {
+      List<AtomicReference<ColumnFamilyHandle>> columnHandles, boolean reset) {
```

Comment:
Will boolean reset better to be part of DBOptions?

Reply:
No. DBOptions are defined for and by RocksDB.
It's better not to reuse them for other purpose.

Because in the original code, RocksDB is only used for Master metastore, and it is always removed/re-created when restart.
But for Dora, we don't want the RocksDB to be removed/re-created again. So, this flag is used so the code can be shared.

Reply:
Oh got it, thinking DBOption is something defined by us

Reply:
"we don't want the RocksDB to be removed/re-created again" is this one settable by end-user?

Reply:
I don't think it should/can be controlled by end-user.
This is for internal use: different components need different usage of the RocksDB store.
For example, the inodestore and blockstore on Master are removed because in current implementation, RocksDB is just a replacement of heapstore. We are not using it as a persistent store. This is not intended to be controlled by end-users.